networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc

```diff
@@ -1,11 +1,56 @@
-[id="learn-about-ovn-kubernetes"]
+[id="about-ovn-kubernetes"]
 = About the OVN-Kubernetes Pod network provider
 include::modules/common-attributes.adoc[]
-:context: learn-about-ovn-kubernetes
+:context: about-ovn-kubernetes
 
 toc::[]
 
-include::modules/nw-ovn-technology-preview.adoc[leveloffset=+1]
+The {product-title} cluster uses a virtualized network for Pod and Service networks.
+The OVN-Kubernetes CNI plug-in is a Pod network provider.
+
+[id="ovn-kubernetes-features"]
+== Features
+
+The OVN-Kubernetes Pod network provider includes the following features:
+
+* Uses Open Virtual Network (OVN) to manage network traffic flows. OVN is a community developed, vendor agnostic network virtualization solution.
+* Implements Kubernetes network policy support, including ingress and egress rules, `podSelector`, and `namespaceSelector`.
+* Implements multicast.
+* Uses the Generic Network Virtualization Encapsulation (GENEVE) overlay network.
+
+[id="network-provider-feature-support"]
+== Network provider feature support
+
+{product-title} offers two choices for the Pod network provider. The following table summarizes the current feature support for both network providers:
+
+.Pod network provider feature comparison
+[cols="50%,25%,25%"]
+|===
+|Feature|OVN-Kubernetes|OpenShift SDN
+
+|Multicast|Supported|Supported
+
+|Kubernetes network policy|Supported {nbsp} (a)|Supported {nbsp} (b)
+
+|Egress firewall {nbsp} (c)|Not supported|Supported
+
+|Egress IPs|Not supported|Supported
+
+|Egress router|Not supported|Supported
+
+|External IP|Not supported|Supported
```

Comment:
drop this row; the "external IP" feature doesn't depend on anything from the network plugin. It will work even with third-party plugins